gateway/1.5/modules/ROOT/nav.adoc

```diff
@@ -80,7 +80,7 @@
 **** xref:policies-pdk-compile-policies.adoc[Compile Custom Policies]
 **** xref:policies-pdk-debug-local.adoc[Debug Custom Policies Locally]
 *** xref:policies-pdk-deploy-policies.adoc[Deploy Custom Policies]
-**** xref:policies-pdk-publish-policies.adoc[Publish Custom Policies to Exchance]
+**** xref:policies-pdk-publish-policies.adoc[Upload Custom Policies to Exchange]
```

Comment:
If you leave the [] blank in xrefs, they get filled in automagically during the build - and you don't have to maintain them manually.